stable/odoo/values.yaml

```diff
@@ -59,6 +59,8 @@ odooEmail: user@example.com
 # smtpPassword:
 # smtpProtocol:
 
+env: {}
```

Comment:
Can you also add some comments and examples there? For example:

```yaml
## Additional environment variables for Odoo
## env:
##  - name: MY_VAR
##    value: my-value
##
env: []
```

Also note that on the README you said the default value was `[]` and here you put `{}`.

Reply:
Fixed